code/index.html

```diff
@@ -2,17 +2,137 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
+    <title>Cherry on Top</title>
     <link rel="stylesheet" href="./style.css" />
+    <link rel="icon" href="favicon.ico" type="image/x-icon">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
     <header>
-      <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
+      <div>
+       <a href="index.html"><img src="./images//icon/icon-pie.png" alt="Cherry on Top" style="float:left; width: 60px; padding: 0 10px;">
+        <h1>Cherry on Top</h1>
+        <h3>Making Life Sweeter One Bite at a Time</h3>
+      </div></a> 
+      <!-- check responsiveness in CSS!! -->
     </header>
```

Comment:
Great that you're using the `header` element! If you open an element inside another element, you should close the descendant element before you close the parent element, like this:
```
<div>
  <a>
    ...
  </a>
</div>
```

instead of
```
<div>
  <a>
</div></a>
```

Reply:
You should keep the CSS to the CSS file. If you want specific styling on this image you could give it a class name:
**HTML**
```
<img
  src="./images//icon/icon-pie.png"
  alt="Cherry on Top"
  class="cherry-logo">
```
**CSS**
```
.cherry-logo {
  float: left;
  width: 60px;
  padding: 0 10px;
}
```